frontend/src/components/ChildrenBigPopupLocation/ChildrenBigPopupLocation.tsx

```diff
@@ -0,0 +1,176 @@
+import { getFiltersForUser } from '../../static/mainFIlters';
+
+type Props = {
+  coordinate: latlngType;
+};
+
+const ChildrenBigPopupLocation = ({ coordinate }: Props) => {
+  const ref = useRef<null | HTMLInputElement>();
+  const [filters, setFilters] = useState('');
+
+  const [locationName, setLocationName] = useState('');
+  const [description, setDescription] = useState('');
+  // eslint-disable-next-line no-unused-vars
+  const [links, setLinks] = useState<string[]>([]);
+  // eslint-disable-next-line no-unused-vars
+  const [files, setFiles] = useState<File[]>([]);
+
+  const proces = process.env.REACT_APP_API_URI;
+
+  const handleChange = (e: any): void => {
+    setLocationName(e.target.value);
+  };
+
+  const handleChangeDescription = (e: any): void => {
+    setDescription(e.target.value);
+  };
+
+  const accessToken = localStorage.getItem('accessToken');
+
+  const onSubmit = async (e: any) => {
+    e.preventDefault();
+    try {
+      const formData = new FormData();
+      formData.append('locationName', locationName);
+      formData.append('description', description);
+      formData.append('coordinates', String(coordinate.lat));
+      formData.append('coordinates', String(coordinate.lng));
+      formData.append('filters', String(filters));
+      formData.append('image', files[0]);
+
+      await axios.post(`${proces}add_personal_location`, formData, {
+        headers: {
+          Authorization: `Bearer ${accessToken}`,
+          'Content-Type': 'multipart/form-data'
+        }
+      });
+
+      setLocationName('');
+      setDescription('');
+      if (ref.current) {
+        ref.current.value = '';
+      }
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  const onChangeAutocomlete = (e: any, value: any) => {
+    setFilters(value);
+  };
+
+  const handleFilesChange = (e: any) => {
+    const { files: filesLst } = e.currentTarget;
+    const filesListArr = [];
+
+    if (filesLst) {
+      for (let i = 0; i < filesLst.length; i += 1) {
+        filesListArr.push(filesLst[i]);
+      }
+
+      setFiles(filesListArr);
+    }
+  };
+
+  const handleFormSubmit = async () => {
+    const formData = new FormData();
+
+    const urls = await Promise.all(
+      files.map(async file => {
+        formData.set('file', file);
+        const res = await fetch(`${process.env.REACT_APP_API_URI}uploadImage`, {
+          method: 'POST',
+          body: formData
+        });
+
+        return res.json();
+      })
+    );
+    setLinks(urls.map(itm => itm.url));
+  };
+
+  return (
+    <form
+      onSubmit={onSubmit}
+      style={{
+        width: '300px',
+        height: '600px',
+        textAlign: 'center',
+        marginTop: '150px'
+      }}
+    >
+      <Typography>Creating location</Typography>
+
+      {/* <img
+        src=""
+        alt=""
+        style={{
+          width: '200px',
+          height: '200px',
+          backgroundColor: 'white'
+        }}
+      /> */}

```

Comment:
delete, if this code not use

Reply:
we should discuss about future functional with that commented code